app/bundles/CoreBundle/Assets/css/app.css

```
@@ -7972,19 +7950,19 @@ ul.line-legend li {
   margin: 0 0 15px 0;
 }
 lesshat-selector {
-  -lh-property: 0; }
+  -lh-property: 0; } 
```

```suggestion
  -lh-property: 0; }
```
The appending space is the only change. So, suggesting a revert.

---

I'm confused about those changes, as the command `grunt compile-less` is the responsible for generating the final CSS file (so it's never manually touched)